### ISSUE TITLE
0 free 0 reserved continuously?

### ISSUE THREAD

**Issue by 2high2die** [kalkih/passport-appointment-bot#42]
Is this normal?
0 free 0 reserved, 1-2 minutes into searching every time. 
I've switched IP's all searches via VPN aswell.
Going all from sessions 1, throttle 0
to sessions 1, throttle 0.5

My friend has the same problem and has given up.

**Comment by xxandeer** [comment]
That is normal since polismyndigheten have implemented captcha on the search page that appears after a few minutes.

**Comment by shsh1801** [comment]
So we do not have a fix for this?

**Comment by kalkih** [comment]
No, there's no fix, the alternative would be to implement the manual captcha solving step again... but the point of this project is kind of lost then, would require manual input every few minutes.

**Comment by kalkih** [comment]
It made sense when there were just two captchas required, one at startup and one to finalize the booking.
